src/couch/src/couch_httpd_auth.erl

```diff
@@ -296,9 +296,9 @@ cookie_authentication_handler(#httpd{mochi_req = MochiReq} = Req, AuthModule) ->
                 end,
             % Verify expiry and hash
             CurrentTime = make_cookie_time(),
+            HashAlgorithms = get_config_hash_algorithms(),
             case chttpd_util:get_chttpd_auth_config("secret") of
                 undefined ->
-                    couch_log:debug("cookie auth secret is not set", []),
```

Comment:
restore this line please.

Reply:
oh yeah I was too quick there while removing my debug output.